app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -51,6 +51,17 @@ def choices_controller?
                               end
     end
 
+    delegate :continuous_applications?, to: :current_application
+
+    def back_link_text
```

Comment:
Should this be in the controller?

Reply:
Looks like it should be in a view helper

Reply:
I tried this at first but the problem is that the view helper doesn't know #any_accepted_offer from the controller and rails raises `undefined method `any_accepted_offer?' for #<ActionView::Base ...`

Reply:
This seems like a bit of a chewing gum vs hair problem. Not going to be solved in this PR